src/design/optForceGAMS/testOptForceSol.m

```diff
@@ -0,0 +1,116 @@
+function [maxGrowthRate, minTarget, maxTarget] = testOptForceSol(model, targetRxn, solution, relax, tol)
+%% DESCRIPTION
```

Comment:
This file (along with any files necessary for input to the function) should be moved to tests/verifiedTests/testOptForceWithGAMS
Otherwise it will not be recognized as a test function, and thus not be run in the test suite (which leads to the code coverage report)